hs_core/tests/api/views/test_get_user_or_group_data.py

```diff
@@ -175,7 +175,7 @@ def test_get_user_data(self):
         response = get_user_or_group_data(request, user_or_group_id=self.john.id, is_group='false')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         resp_json = json.loads(response.content)
-        self.assertEqual(resp_json['name'], 'John K Clarson')
+        self.assertEqual(resp_json['name'], 'Clarson, John K')
```

Comment:
We should still have some tests that do not have the comma in them. (Maybe there are more tests in the file, I haven't opened it, but it looks like you updated all of them.)